src/main/java/io/fabric8/jenkins/openshiftsync/PodTemplateUtils.java

```diff
@@ -423,7 +424,8 @@ protected static boolean configMapContainsSlave(ConfigMap configMap) {
 
     protected static boolean hasSlaveLabelOrAnnotation(Map<String, String> map) {
         return map != null && map.containsKey(IMAGESTREAM_AGENT_LABEL)
-                && map.get(IMAGESTREAM_AGENT_LABEL).equals(IMAGESTREAM_AGENT_LABEL_VALUE);
+                && (map.get(IMAGESTREAM_AGENT_LABEL).equals(IMAGESTREAM_AGENT_LABEL_VALUE) ||
+                        map.get(IMAGESTREAM_AGENT_LABEL).equals(IMAGESTREAM_AGENT_ALTERNATE_LABEL_VALUE));
```

Comment:
I forget how order of precedence exactly works here, but you might need to encapsulate within parenthesis the or check on the two possible label values 

i.e. isolate them from the and check with the map null check and presence of the label

Reply:
yeah ignore this / mark resolved